packages/blockchain-wallet-v4/src/redux/wallet/sagas.ts

```diff
@@ -326,7 +326,7 @@ export default ({ api, networks }) => {
     const wallet = yield select(S.getWallet)
     const accounts = Wallet.selectHDAccounts(wallet)
     const accountsWithMissingDerivations = accounts
-      .filter((acct) => acct.derivations.size === 0)
+      .filter((acct) => acct.derivations.size !== 2)
```

Comment:
@plondon should we do `acct.derivations.size < DERIVATION_LIST.length` like you first proposed?